readme.md

```diff
@@ -187,32 +200,32 @@ The invocation may be immediate or it might happen later. In the example above,
 
 > We can also pass in anonymous functions (i.e., define a nameless function directly inside of the event listener) instead of previously-defined functions.
 
-### You Do: Practice (10 minutes / 0:35)
+### You Do: Practice (2:55 - 3:05, 10 minutes)
 
 Visit this [repository](https://github.com/ga-wdi-exercises/event-listener-practice.git) and follow the instructions.
 
-### Before We Go On... (5 minutes / 0:40)
+### Before We Go On... (3:05 - 3:10, 5 minutes)
 
 Usually when we do anything with functions, we put parentheses after the function name. Here, we have `handleClickEvent` without any parens.
 
 Try adding parentheses at the end of this line:
 
 ```js
-button.on("click", handleClickEvent())
+button.on("click", handleClickEvent() );
 ```
 
 Refresh your page. What was different? Why?
 
 > You'll notice that "I was clicked!" pops up immediately upon reload. Also note that the event while it does fire, isn't doing anything. When we include `()` we invoke the function expression. Without the `()`, we're using the function expression as a reference.
 
 
-## Break (10 minutes / 0:50)
+## Break (3:20 - 3:30, 10 minutes)
 
-### You Do: Color Scheme Switcher (30 minutes / 1:20)
+### You Do: TimerJS (3:30 - 4:00, 30 minutes)
```

Comment:
I'm not entirely sure if I like Timer JS better than color scheme. I think starting out, color scheme switcher is easier and honestly I think more valuable then timeJS